python-threatexchange/threatexchange/extensions/vpdq/video_vpdq.py

```diff
@@ -112,3 +117,7 @@ def get_examples() -> t.List[str]:
             timestamp += 1.0
             frame_number += 1
         return [vpdq_to_json(VPDQ_features)]
+
+
+class VPDQHashIndex(VPDQIndex):
+    _SIGNAL_TYPE = VideoVPDQSignal
```

Comment:
blocking: Why do you need this wrapper class?

Reply:
Removed it! Was confused by "_SIGNAL_TYPE".